docs/source/iam-group.rst

```diff
@@ -164,5 +164,13 @@ modify, and destroy all of the BinaryAlert infrastructure:
       policy_arn = "${aws_iam_policy.binaryalert_admin.arn}"
     }
 
+To authenticate with environment variables set the following:
+
+.. code-block:: bash
+
+  export AWS_ACCESS_KEY_ID="an-access-key"
+  export AWS_SECRET_ACCESS_KEY="a-secret-key"
+  export AWS_DEFAULT_REGION="us-west-2"
```

Comment:
It's often easier to just use a [named profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html), then all you have to do is `export AWS_PROFILE=binaryalert`

If you could that, too, that'd be great

Reply:
Good idea. I added a note in f5191a9.